thefuck/rules/upper_to_lower_case.py

```diff
@@ -0,0 +1,54 @@
+import subprocess
+import itertools
+
+
+def find_all_possible_commands(command):
+    lower_command = command.script.lower().split()
+    upper_command = command.script.upper().split()
+    all_commands = []
+    for i in lower_command:
+        all_commands.append(i)
+    for i in upper_command:
+        all_commands.append(i)
+    permutations = itertools.permutations(all_commands, len(lower_command))
+    new_commands = []
+    for per in permutations:
+        new_command = ""
+        for word in per:
+            new_command += word + " "
+        new_command = new_command.strip()
+        if new_command not in new_commands:
+            new_commands.append(new_command)
+    return new_commands
+
+
+def find_all_correct_commands(new_commands : list):
+    all_correct_commands = []
+    for com in new_commands:
+        result = subprocess.getstatusoutput(com)
```

Comment:
I don't think we want to actually run the command, this seems really dangerous!  What if the command is autocorrected to be an 'rm' for instance?

Reply:
Do you know how am I going to check if a command is valid or not? (without running it)

Reply:
Yes this is very dangerous! I agree.